File: src/main/java/com/ptah/entity/acceptance/Acceptance.java
package com.ptah.entity.acceptance;

import com.ptah.common.BaseEntity;
import com.ptah.entity.project.Project;
import jakarta.persistence.ManyToOne;

public class Acceptance extends BaseEntity {
    private String name;
    @ManyToOne
    private Project project;
    private String comments;
    private AcceptStatus status;
}

File: src/main/kotlin/com/ptah/entity/acceptance/Acceptance.kt
package com.ptah.entity.acceptance

import com.ptah.common.BaseEntity
import com.ptah.entity.project.Project
import jakarta.persistence.ManyToOne

class Acceptance : BaseEntity() {
    private val name: String? = null

    @ManyToOne
    private val project: Project? = null
    private val comments: String? = null
    private val status: AcceptStatus? = null
}
